### ISSUE TITLE
Reformer for questions answering(squad)

### ISSUE THREAD

**Issue by bela572** [huggingface/transformers#11712]
I want to use Reformer for Questions Answering.  I tried to use pretrained model 'google/reformer-crime-and-punishment'. I was using this example https://huggingface.co/transformers/custom_datasets.html#qa-squad just replaced with reformer. I get the exception related with pad, cls tokens, sequence length and so on, but that does not matter now, I first want to know: is it even possible to get good results(model to answer questions more than 50 percent accuracy for example? Because I see that you working on this model now and some functions maybe are not implemented yet or so. I saw issues like this: https://github.com/huggingface/transformers/issues/5436 and you wrote that you be very surprised if it will give any good results using reformer for q&a. (but it was year ago, so maybe it changed)
Thanks

**Comment by cronoik** [comment]
It hasn't changed yet. The problem is not code, the problem is the lack of a decent pre-trained model.
